qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -754,6 +761,17 @@ def _inject_userscripts(self):
                                    .format(new_script.name()))
             scripts.insert(new_script)
 
+    def _init_focustools(self):
```

Comment:
I think it'd make sense to separate reading `focustools.js` and calling `focustools.load` here. Then the file can just be read with the others in `_init_js` instead of having its own initialization.

Reply:
I had some issues with the js getting called out of order (the load call seemed to be called before focustools initialized), so I think I'm going to keep it as is for now, if you have any ideas on avoiding this, let me know: https://travis-ci.org/qutebrowser/qutebrowser/jobs/359673240